tests/fe/br_approximation_01.cc

```diff
@@ -0,0 +1,518 @@
+// ---------------------------------------------------------------------
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+// ---------------------------------------------------------------------
+
+
+// projects the shape functions from the Bernardi-Raugel elements
+// into different polynomial spaces and computes the error of the
+// approximation. The error should be close to machine precision
+// for polynomials of degree 2 or higher
```

Comment:
This description doesn't seem right: doesn't this element only implement one degree?

Reply:
Yes the degrees are for the polynomial spaces that are being projected into. Since the Bernardi-Raugel polynomials have polynomial degree 2 in each variable, any projections into FE_Q<dim>(deg) with deg>=2 should be exact. Should I rephrase that in the documentation?